python/agents/polymarket/gamma.py

```
@@ -0,0 +1,188 @@
+                return data
+            else:
+                markets: list[Market] = []
+                for market_object in data:
+                    markets.append(self.parse_pydantic_market(market_object))
+                return markets
+        else:
+            print(f"Error response returned from api: HTTP {response.status_code}")
+            raise Exception()
+
+    def get_events(
+        self, querystring_params={}, parse_pydantic=False, local_file_path=None
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)